spec.html

```diff
@@ -27921,8 +28176,9 @@ <h1>
         </dl>
 
         <emu-alg>
-          1. Assert: Exactly one element of _referrer_.[[LoadedModules]] is a Record whose [[Specifier]] is _specifier_, since LoadRequestedModules has completed successfully on _referrer_ prior to invoking this abstract operation.
-          1. Let _record_ be the Record in _referrer_.[[LoadedModules]] whose [[Specifier]] is _specifier_.
+          1. Let _records_ be a List consisting of each LoadedModuleRequest Record _r_ of _referrer_.[[LoadedModules]] such that ModuleRequestsEqual(_r_, _request_) is *true*.
```

Comment:
@tc39/ecma262-editors ecmarkup doesn't like this way of defining `_r_`. What is your preferred wording? https://github.com/tc39/ecma262/pull/3057#discussion_r1244354877 suggests changed but doesn't fix the linting problem.

Reply:
This wording looks fine to me - ecmarkup can recognize similar patterns, but its regexps aren't fancy enough for this precise wording. I would use the [`[declared="r"]`](https://tc39.es/ecmarkup/#algorithm-step-annotation) escape hatch here.